### PR TITLE
added disqus to blog

### PR DESCRIPTION
**Pull request by matthewl20**

### CODE REVIEW

**Comment by matthewl20** [comment]
@halloffame @rift137 Could one of you take a look at this when you have a moment?

File: app/views/articles/templates/default/_article_author.html.slim [hunk 1]
```diff
@@ -7,4 +7,4 @@ hr
     .item-body
       h3.name = link_to author.name, author_path(author)
       - if profile = article_author_profile(author).presence
-        == profile.body
+        == profile.body
```
Comment (rift137) [inline app/views/articles/templates/default/_article_author.html.slim]: Gotta add dem end lines

File: app/views/articles/templates/default/_article_body.html.slim [hunk 2]
```diff
@@ -21,3 +20,18 @@
...
+
+        console.log("ID: #{@article.id}")
+      };
+
+      (function() { // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        s.src = "https://#{@site.presentation_data[:disqus_subdomain]}.disqus.com/embed.js";
+        s.setAttribute('data-timestamp', +new Date());
+        (d.head || d.body).appendChild(s);
+      })();
```
Comment (rift137) [inline app/views/articles/templates/default/_article_body.html.slim]: end line here too

File: app/views/articles/templates/default/_article_body.html.slim [hunk 3]
```diff
@@ -21,3 +20,18 @@
         enabled: true
       }
     });
+  - if @site.presentation_data[:disqus_subdomain].present?
```
Comment (rift137) [inline app/views/articles/templates/default/_article_body.html.slim]: remove this

File: app/views/articles/templates/default/index.html.slim [hunk 4]
```diff
@@ -9,5 +9,25 @@
...
+        this.page.identifier = "#{@article.id}";
+      };
+
+      (function() { // DON'T EDIT BELOW THIS LINE
+        var d = document, s = d.createElement('script');
+        s.src = "//#{@site.presentation_data[:disqus_subdomain]}.disqus.com/embed.js";
+        s.setAttribute('data-timestamp', +new Date());
+        (d.head || d.body).appendChild(s);
+      })();
+    script id="dsq-count-scr" src="//#{@site.presentation_data[:disqus_subdomain]}.disqus.com/count.js" async=""
```
Comment (rift137) [inline app/views/articles/templates/default/index.html.slim]: end line

**Pull request merged**